desktop/app/ctl.js

```diff
@@ -0,0 +1,8 @@
+import {keybaseBinPath} from './paths'
+import exec from './exec'
+
+export function ctlStop () {
+  const binPath = keybaseBinPath()
+  const args = ['ctl', 'stop']
+  exec(binPath, args, 'darwin', 'prod', function () {})
```

Comment:
Now it's being used, on quit, after `ctlStop` completes we do `app.quit()`.

I added a `no-wait` option to ctl stop so it happens very fast.